tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -1290,37 +1295,46 @@ public static MissingStatus getMissingStatus(CLDRFile sourceFile, String path, b
-            && !sourceLocale.equals(sourceLocaleID)) {
-            int debug = 0;
+        if (DEBUG) {
+            if (path.equals("//ldml/characterLabels/characterLabelPattern[@type=\"subscript\"]")) {
+                int debug = 0;
+            }
+            if (!isAliased && !sourceLocale.equals(sourceLocaleID)) {
+                int debug = 0;
+            }
         }
 
         if (value == null) {
-            result = ValuePathStatus.isMissingOk(sourceFile, path, latin, isAliased) ? MissingStatus.MISSING_OK : MissingStatus.ABSENT;
+            result = ValuePathStatus.isMissingOk(sourceFile, path, latin, isAliased) ? MissingStatus.MISSING_OK
+                : MissingStatus.ABSENT;
         } else {
             /*
              * skipInheritanceMarker must be false for getSourceLocaleIdExtended here, since INHERITANCE_MARKER
              * may be found if there are votes for inheritance, in which case we must not skip up to "root" and
              * treat the item as missing. Reference: https://unicode.org/cldr/trac/ticket/11765
              */
             String localeFound = sourceFile.getSourceLocaleIdExtended(path, status, false /* skipInheritanceMarker */);
+            final boolean localeFoundIsRootOrCodeFallback = localeFound.equals("root")
+                || localeFound.equals(XMLSource.CODE_FALLBACK_ID);
+            final boolean isParentRoot = CLDRLocale.getInstance(sourceFile.getLocaleID()).isParentRoot();
             /*
              * Only count it as missing IF the (localeFound is root or codeFallback)
              * AND the aliasing didn't change the path.
              * Note that localeFound will be where an item with ↑↑↑ was found even though
              * the resolved value is actually inherited from somewhere else.
              */
-            if (localeFound.equals("root")
-                || localeFound.equals(XMLSource.CODE_FALLBACK_ID)) {
-                result = ValuePathStatus.isMissingOk(sourceFile, path, latin, isAliased)
-                    || sourceLocaleID.equals("en")
-                    ? MissingStatus.ROOT_OK
-                        : MissingStatus.ABSENT;
-            } else if (isAliased) {
-                result = MissingStatus.ALIASED;
-            } else {
+
+            if (localeFoundIsRootOrCodeFallback) {
+                result = ValuePathStatus.isMissingOk(sourceFile, path, latin, isAliased) ? MissingStatus.ROOT_OK
+                    : isParentRoot ? MissingStatus.ABSENT
+                        : MissingStatus.ALIASED;
+            } else if (!isAliased) {
                 result = MissingStatus.PRESENT;
+            } else if (isParentRoot) { // We handle ALIASED specially, depending on whether the parent is root or not.
+                result = ValuePathStatus.isMissingOk(sourceFile, path, latin, isAliased) ? MissingStatus.MISSING_OK
+                    : MissingStatus.ABSENT;
+            } else {
+                result = MissingStatus.ALIASED;
             }
```

Comment:
Maybe related to https://unicode-org.atlassian.net/browse/CLDR-14827

Reply:
This does appear to be the problematic area. In the test case I'm using, isAliased and isParentRoot are both true.

The old code would have first tested isAliased, and finding it to be true, it would have returned MissingStatus.ALIASED.

The new code does not handle the case of isAliased being true until after it returns ABSENT on the basis of isParentRoot and isMissingOk.